### ISSUE TITLE
2021-09-30_63641_TSE.tar results

### ISSUE THREAD

**Issue by nils-tekampe** [konfidas/TTC#106]
While checking https://github.com/konfidas/TTC/blob/main/testdata/positive/2021-09-30_63641_TSE.tar the following errors have been reported 

<p> Dieser Report deckt die folgenden Log Message Archive ab:</p>
    2021-09-30_63641_TSE.tar
Es wurden keine Fehler bei der Erstellung des Gesamtarchivs gefunden. Es können aber Fehler in einzelnen Logs aufgetreten sein.Die LogMeasse Unixt_1627475722_Sig-6_Log-Sys_configureLogging.log wurde gelesen. Dabei wurden 2 Fehler gefunden.
Das Parsen der Message Unixt_1627475722_Sig-6_Log-Sys_configureLogging.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Das Parsen der Message Unixt_1627475722_Sig-6_Log-Sys_configureLogging.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld parameters startet nicht mit einer Sequenz..
Die LogMeasse Unixt_1627475722_Sig-7_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1627475722_Sig-7_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1627902745_Sig-26_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1627902745_Sig-26_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1629286469_Sig-93_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1629286469_Sig-93_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1629286469_Sig-95_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1629286469_Sig-95_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1629286469_Sig-96_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1629286469_Sig-96_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1629286469_Sig-97_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1629286469_Sig-97_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-155_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-155_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-157_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-157_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-158_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-158_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-159_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-159_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-160_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-160_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632148684_Sig-161_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632148684_Sig-161_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Die LogMeasse Unixt_1632908000_Sig-199_Log-Sys_selfTest.log wurde gelesen. Dabei wurden 1 Fehler gefunden.
Das Parsen der Message Unixt_1632908000_Sig-199_Log-Sys_selfTest.log ist fehlgeschlagen mit foglendem Fehler Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld result wurde nicht gefunden...
Um diesen Bericht zu generieren, wurden die folgenden Validatoren verwndet:
    class de.konfidas.ttc.validation.CertificateFileNameValidator
    class de.konfidas.ttc.validation.TimeStampValidator
    class de.konfidas.ttc.validation.SignatureCounterValidator
    class de.konfidas.ttc.validation.LogMessageSignatureValidator
Während der Validierung wurden 7 Fehler gefunden.
Die folgenden Probleme wurden gefunden, sind aber nicht direkt auf eine Log Message bezogen: 
    Abweichung des Dateinamens vom Subject. Es wurde E11E1A155B6166B2F1844E8F063AE44837869BFA5689DFB7BAE8524444D54E52 gefunden aber BSI-DSZ-CC-1153_BSI-DSZ-CC-1130_E74F08931CA94A5EA087118D5BFAD425 erwartet.Die folgenden Fehler wurden spezifisch für Log Messages gefunden:
(gültige Log Nachrichten werden in diesem Bericht nicht gelistet)
    Unixt_1627475722_Sig-4_Log-Sys_registerClient.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 1. Der nächste, gefundene Signaturzähler ist 4.
    Unixt_1627902745_Sig-26_Log-Sys_selfTest.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 22. Der nächste, gefundene Signaturzähler ist 26.
    Unixt_1629286469_Sig-93_Log-Sys_selfTest.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 78. Der nächste, gefundene Signaturzähler ist 93.
    Unixt_1632138925_Sig-154_Log-Sys_updateTime.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 117. Der nächste, gefundene Signaturzähler ist 154.
    Unixt_1632908000_Sig-199_Log-Sys_selfTest.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 192. Der nächste, gefundene Signaturzähler ist 199.
    Unixt_1632999742_Sig-205_Log-Tra_No-50_Start_Client-1063641.log:
        Für die TSE e11e1a155b6166b2f1844e8f063ae44837869bfa5689dfb7bae8524444d54e52 fehlt der Signaturzäher 204. Der nächste, gefundene Signaturzähler ist 205.

We need to check whether these are actual errors 

Detailed results:
[results.txt](https://github.com/konfidas/TTC/files/7799002/results.txt)

**Comment by maximsk** [comment]
`selfTest` and `configureLogging` are fixed in https://github.com/konfidas/TTC/pull/151

**Comment by nils-tekampe** [comment]
True. However, the archive still produces errors as 4 signature counters are missing. After shortly looking into the archive, these seem to be true errors
